README.md

```diff
@@ -14,7 +14,9 @@ An impact assessment is due when
 
 The procedure is as follows:
 
-1. Rebase changes, mandated by your proposal, atop of `ghc-9.10` branch.
+1. Rebase changes, mandated by your proposal, atop of `ghc-9.10.2` tag. Mind: the branch `ghc-9.10` is not okay since different minor versions
```

Comment:
Could we express it in a way, which would require less updates in the future, when Stackage moves to newer versions of GHC?